src/bika/lims/browser/worksheet/tools.py

```diff
@@ -78,3 +82,13 @@ def showRejectionMessage(worksheet):
                 "worksheet at ${ws_id}",
                 mapping={'ws_id':_ws.getId()})
         worksheet.plone_utils.addPortalMessage(msg)
+
+
+def getWorksheetLayouts():
+    """ Getting additional layouts for Worksheet
+    """
+    layouts = ()
+    for name, layout_utility in getUtilitiesFor(IWorksheetLayouts):
+        layouts += layout_utility.getLayoutsList()
```

Comment:
Maybe better make `layouts` a list and use the `append` method for each entry the utility/adapter returns. Might cause an `TypeError: can only concatenate tuple (not "list") to tuple` otherwise.